packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -16,7 +16,7 @@ def test_if_marathon_app_can_be_deployed(cluster):
     the one assigned to test - test succeds.
     """
     app, test_uuid = cluster.get_test_app()
-    cluster.deploy_test_app_and_check(app, test_uuid)
+    cluster.deploy_test_app_and_check(app, test_uuid, expected_user=app['user'])
```

Comment:
can we default / if `expected_user` is None, then use app['user']? Then I think this diff just goes to adding that default lookup, adding the default user, and adding the one extra line to change user for the exceptional mesos containerizer case.